server/backend/api/app/jwt_validation.py

```diff
@@ -215,13 +215,18 @@ def authorize_by_app_id(
             headers={"WWW-Authenticate": "Bearer"},
         )
 
-    required_group = f"{application.application_name.upper()}_ACCESS_ADMIN"
-    groups = claims[JWT_GROUPS_KEY]
+    required_role = f"{application.application_name.upper()}_ACCESS_ADMIN"
+    access_roles = get_access_roles(claims)
```

Comment:
I don't understand what's going on entirely. Authorize by app id - this is just for the FAM API backend used by the FAM API, right, so we only care about X_ACCESS_ADMIN in the context of a specific app...

I guess it isn't clear on how this authorize function differs from the prior one - maybe we need some clarifying comments on these functions.

Reply:
Yup, it was clear for me, but when look into implementation I understand we do (for now) only care X_(ACCESS_ADMIN) at that "X" application. I had short question/comment on ticket #359 using hardcoded "_ACCESS_ADMIN" as logic for the confusion. I do need to clarify a bit, but I think in post mvp Conrad have some idea or plan for this to change.

I am not sure what 'prior one' you might referring to, here the change is simply rename variables from 'groups' to 'roles'.

Reply:
Hope this is clear now for you Basil.